templates/emails/group.monthlyreport.hbs

```diff
@@ -117,31 +117,36 @@ th {
 <h2>Changelog</h2>
 
 <ul>
-<li><b>Request Money:</b> 
-Sometimes you want to send a backer or a sponsor a specific link for a donation, or you want to do a special subscription deal or sell tickets to a meetup. We released a new feature to generate a specific URL for this and you can find it in your transactions page:
-<center><img style="width:100%;" src="https://cl.ly/2i1o370V3C3S/Screen%20Shot%202017-02-06%20at%2011.21.13%20PM.png" /></center>
+<li><b>Sell Tickets:</b> 
+We are rolling out the first release of a feature to sell event tickets on your Open Collective page! It's a great way to raise funds for your collective. Ping us if you want to test it! (just reply to this email) <a href="http://opencollective.com/brusselstogether/events/meetup-2?{{utm}}">http://opencollective.com/brusselstogether/events/meetup-2</a>
+<center><a href="http://opencollective.com/brusselstogether/events/meetup-2?{{utm}}"><img style="width:100%;" src="https://cl.ly/081v0t2p2e3Q/Screen%20Shot%202017-03-03%20at%2010.38.18%20AM.png" /></a></center>
 </li> 
-<li><b>We’ve changed the way your backers are charged:</b> 
-We now charge recurring monthly subscriptions on 1st of every month (only for new recurring donations).
-<p>The goal is to help you better manage your budget. You will now start the month with a minimum budget that will get improved with any new donations. This will also enable us in the future to charge a single payment for multiple monthly donations from the same user and reduce transaction fees.</p>
+<li><b>npm-postinstall:</b> 
+A module to prompt your users to donate to your collective after `npm install` <a href="https://github.com/opencollective/opencollective-postinstall">https://github.com/opencollective/opencollective-postinstall</a>. 
+There has been a <a href="https://twitter.com/_developit/status/836377502939557888">lively conversation on twitter</a> about it. We are looking for someone to try it. Are you interested?
+<p><center><a href="https://github.com/opencollective/opencollective-postinstall"><img style="width:100%;" src="https://cl.ly/323M3x2Y1J3E/Screen%20Shot%202017-02-22%20at%202.03.58%20AM.png" /></a></center></p>
+</li>
+<li>
+  <b>Backer/Sponsor Widget:</b> If you are supporting a (or many!) collectives, you can <a href="https://github.com/OpenCollective/OpenCollective/wiki/Website-widget">add a banner to your website</a> with to showcase them.
```

Comment:
"supporting a collective (or many collectives)", otherwise "a (or many!) collectives" mixing singular and plural.